aten/src/ATen/native/ConstantPadNd.cpp

```diff
@@ -54,7 +54,18 @@ Tensor constant_pad_nd(const Tensor& self, IntArrayRef pad, Scalar value) {
         new_shape.emplace_back(new_dim);
     }
 
-    auto output = at::empty(new_shape, self.options());
+    at::Tensor output;
+    if (self.is_quantized()) {
+        const auto qscheme = self.qscheme();
+        TORCH_CHECK(qscheme == kPerTensorAffine || qscheme == kPerTensorSymmetric,
+                    "Only per-tensor padding is supported.");
+        const auto memory_format = self.suggest_memory_format();
+        output = at::_empty_affine_quantized(
+            new_shape, self.options().memory_format(memory_format),
+            self.q_scale(), self.q_zero_point(), c10::nullopt);
```

Comment:
This is required because if not explicitly specified as `nullopt` there will be two conflicting memory_format arguments -- one from the options and one as an argument.